modin/experimental/xgboost/xgboost.py

```diff
@@ -76,6 +117,200 @@ def __iter__(self):
+        """
+        Set feature types (column types).
+
+        This is for displaying the results and unrelated
+        to the learning process.
+
+        Parameters
+        ----------
+        feature_types : list or None
+            Labels for features. None will reset existing feature names
+
+        Returns
+        -------
+        Add in dmatrix new value of parameter ``feature_types``.
+        """
+        if feature_types is not None:
+            if not isinstance(feature_types, (list, str)):
+                raise TypeError("feature_types must be string or list of strings")
+            if isinstance(feature_types, string_types):
+                # single string will be applied to all columns
+                feature_types = [feature_types] * self.num_col()
+            try:
+                if not isinstance(feature_types, str):
+                    feature_types = list(feature_types)
+                else:
+                    feature_types = [feature_types]
+            except TypeError:
+                feature_types = [feature_types]
+        else:
+            feature_types = None
+        self._feature_types = feature_types
+
+    def num_row(self):
+        """
+        Get the number of rows in the DMatrix.
+
+        Returns
+        -------
+        number of rows : int
+        """
+        return self.rows
+
+    def num_col(self):
+        """
+        Get the number of columns (features) in the DMatrix.
+
+        Returns
+        -------
+        number of columns : int
+        """
+        return self.cols
+
+    def get_float_info(self, name):
+        """
+        Get float property from the DMatrix.
+
+        Parameters
+        ----------
+        name: str
+            The field name of the information
+
+        Returns
+        -------
+        info : array
+            a numpy array of float information of the data
+        """
+        code = "self.{}".format(name)
+        return eval(code)
+
+    def set_info(
+        self,
+        *,
+        label=None,
+        feature_names=None,
+        feature_types=None,
+        feature_weights=None,
+    ) -> None:
+        """
+        Set meta info for DMatrix.
+
+        Parameters
+        ----------
+        label : modin.pandas.DataFrame or modin.pandas.Series
+            Labels used for training.
+        missing : float
+            Value in the input data which needs to be present as a missing
+            value. If None, defaults to np.nan.
+        silent : boolean
+            Whether print messages during construction
+        feature_names : list
+            Set names for features.
+        feature_types : list
+            Set types for features.
+        feature_weights : array_like
+            Set feature weights for column sampling.
+        enable_categorical: boolean
+            Experimental support of specializing for categorical features.
+
+        Returns
+        -------
+        Set meta info for DMatrix.
```

Comment:
This method doesn't return any values. `Returns` section could be omitted

Reply:
This was delete